doc/source/cluster/kubernetes/examples/rayjob-kueue-priority-scheduling.md

```diff
@@ -0,0 +1,317 @@
+  name: prod-priority
+value: 1000
+description: "Priority class for prod jobs"
+---
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: WorkloadPriorityClass
+metadata:
+  name: dev-priority
+value: 100
+description: "Priority class for development jobs"
+```
+
+Create the Kueue resources:
+```bash
+kubectl apply -f kueue-resources.yaml
+```
+
+## Deploy a RayJob
+
+Download the RayJob that that will execute all the steps documented in [Fine-tune a PyTorch Lightning Text Classifier](https://docs.ray.io/en/master/train/examples/lightning/lightning_cola_advanced.html). The [source code](https://github.com/ray-project/kuberay/tree/master/ray-operator/config/samples/pytorch-text-classifier) is also in the kuberay repo.
+
+```bash
+curl -LO https://raw.githubusercontent.com/ray-project/kuberay/master/ray-operator/config/samples/pytorch-text-classifier/ray-job.pytorch-distributed-training.yaml
+```
+
+Before creating the RayJob, modify the RayJob metadata with:
+* a label to assign the RayJob to the LocalQueue created earlier.
+* a label to assign the RayJob with the `dev-priority` priority class.
+* Modify the name to indicate this is a job for development purposes.
+
+```yaml
+metadata:
+  generateName: dev-pytorch-text-classifier-
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+    kueue.x-k8s.io/priority-class: dev-priority
+```
+
+Also note the resources required for this RayJob by looking at the resources requested by the nodes:
+```yaml
+resources:
+  limits:
+    cpu: "8"
+    memory: "20G"
+    nvidia.com/gpu: "1"
+  requests:
+    cpu: "8"
+    memory: "20G"
+    nvidia.com/gpu: "1"
+```
+
+Now deploy the RayJob:
+```bash
+$ kubectl create -f ray-job.pytorch-distributed-training.yaml
+rayjob.ray.io/dev-pytorch-text-classifier-r6d4p created
+```
+
+Verify that the RayCluster and Ray job are running:
+```bash
+$ kubectl get pod
+NAME                                                      READY   STATUS    RESTARTS   AGE
+dev-pytorch-text-classifier-r6d4p-4nczg                   1/1     Running   0          4s
+torch-text-classifier-r6d4p-raycluster-br45j-head-8bbwt   1/1     Running   0          34s
+```
+
+### Queuing Multiple RayJobs
+
+You can create multiple RayJobs to see how Kueue interacts with Kuberay to implement queueing.
+```bash
+$ kubectl create -f ray-job.pytorch-distributed-training.yaml
+rayjob.ray.io/dev-pytorch-text-classifier-8vg2c created
+$ kubectl create -f ray-job.pytorch-distributed-training.yaml
+rayjob.ray.io/dev-pytorch-text-classifier-n5k89 created
+$ kubectl create -f ray-job.pytorch-distributed-training.yaml
+rayjob.ray.io/dev-pytorch-text-classifier-ftcs9 created
+```
+
+Because our `ClusterQueue` was configured with only quotas for 1 RayJob, Kueue will automatically suspend new RayJobs
+until there is sufficient quotas available.
+
+You can also inspect the `ClusterQueue` to see available / used quotas:
+```bash
+$ kubectl get clusterqueue
+NAME            COHORT   PENDING WORKLOADS
+cluster-queue            5
```

Comment:
We should make the result more predictable so that users can verify whether they are on the right track or not. That's why I decide to change "multiple" to "3". See https://github.com/ray-project/ray/pull/42903/files#r1476909906 for more details.

```suggestion
cluster-queue            2
```

Reply:
done